armi/apps.py

```diff
@@ -223,6 +231,76 @@ def getParamRenames(self) -> Dict[str, str]:
             self._paramRenames = renames, self._pm.counter
         return renames
 
+    def registerUserPlugins(self, pluginPaths):
+        """
+        Register additional plugins passed in by importable paths.
```

Comment:
docstring should have blank line after short summary line https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard

Reply:
Done!